lib/index.js

```diff
@@ -246,6 +279,82 @@ export class Baker extends EventEmitter {
     }
   }
 
+  /**
+   * Prepares a function to process crosswalk image paths.
+   * @param {string} crosswalkPath - The base path for crosswalk images.
+   * @returns {Function} A function that maps over data entries to update image paths.
+   */
+  prepareCrosswalk(/** @type {string} */ crosswalkPath) {
+    /**
+     * Processes data entries to set appropriate paths for images.
+     * @param {Object.<string, string>[]} data - Array of objects, each containing image references.
+     * @returns {Object.<string, string>[]} The data with updated image paths.
+     */
+    return function setCrosswalkPath(
+      /** @type {Object.<string, string>[]} */ data
+    ) {
+      /** @type {Object.<string, string>[]} */ data
+      const isAbsolutePath =
+        isProductionEnv &&
+        process.env.BAKER_BASE_PATH &&
```

Comment:
make sure env vars exist and throw an error if they dont. Should probably make a helper function for this since we need to do it in several places

Reply:
like the issue above, if we make that check it will break local dev